opendkim/opendkim.c

```diff
@@ -11656,8 +11656,8 @@ mlfi_eoh(SMFICTX *ctx)
 	    (status != 0 || user == NULL || domain == NULL ||
 	     user[0] == '\0' || domain[0] == '\0'))
 	{
-		strlcpy(addr, conf->conf_defsender, sizeof addr);
-		status = dkim_mail_parse(addr, &user, &domain);
+		strlcpy(dkimf_dstring_get(addr), conf->conf_defsender, sizeof addr);
```

Comment:
It seems it should use `dkim_dstring_copy()` instead of `strlcpy()`.

Reply:
Probably, I changed it in an additional commit. Thanks.

Reply:
I confirmed the commit,  and it looks good for me.  I'm sorry for the typo on function name, and thank you for the commit.

I'll merge this to my branch.